resolwe_bio/processes/expression/cuffnorm.yml

```diff
@@ -104,20 +104,23 @@
       experiment='{% for experiment in cuffquant %}{{ experiment.cxb.file }},{% endfor %}'
       replicates='{% for r in replicates %}{{r}},{% endfor %}'
       labels='{% for l in labels %}{{l}},{% endfor %}'
+      sample_names='{% for sample in cuffquant %}{{sample|sample_name}},{% endfor %}'
 
       labels=${labels%?}
       experiment=${experiment%?}
       replicates=${replicates%?}
+      sample_names=${sample_names%?}
 
       experiments=`cufflinks_sample_groups.py ${experiment} ${replicates}`
 
       re-progress 0.1
 
       cuffnorm --output-dir ./ --num-threads '{{ threads }}' --labels ${labels} {% if annotation.gtf.file %} '{{ annotation.gtf.file }}' {% elif annotation.gff.file %} '{{ annotation.gff.file }}' {% elif annotation.merged_gtf.file %} '{{ annotation.merged_gtf.file }}' {% endif %} ${experiments}
-
+      
```

Comment:
Remove spaces. Consider installing plugin for your editor that do this automatically.